exec.c

```diff
@@ -41,3 +47,40 @@ int executeCommand(char **argv)
 
 	return (0);
 }
+
+/**
+ * handlePath - returns fullpath of command passed
+ * @cmd: command passed
+ *
+ * Return - return fullpath of command passed
+ */
+char *handlePath(char *cmd)
+{
+	char *path = _getenv("PATH");
+	char *full_path = malloc(sizeof(char) * 1024);
+	char *path_copy, *token;
+
+	path_copy = _strdup(path);
+	if ((path == NULL) || (path_copy == NULL))
+	{
+		perror("PATH not found");
+		return (NULL);
+	}
+
+	token = _strtok(path_copy, ":");
+	while (token)
+	{
+		/*concatenate the token in PATH with the command passed*/
+		snprintf(full_path, 1024, "%s/%s", token, cmd);
```

Comment:
Can you replace this function with a `_strcat()` or a custom snprintf?

Reply:
I'll check it out. My system is down now. I think I'll check the strcat